docs/Glossary.md

```diff
@@ -1,3 +1,59 @@
 # Glossary
 
-See [Terminologies](/introduction/Terminologies.html).
+## App(object specifications)
```

Comment:
isn't this a duplicate of https://github.com/Travix-International/frint/blob/master/docs/introduction/Terminologies.md?